src/index.html

```diff
@@ -17,6 +17,85 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        href="index.html"
+        class="logo_link"
+      >
+        <img
+          src="images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="li">
+            <a
+              href="index.html"
+              class="nav_link is-active"
+            >
+              Apple
+            </a>
```

Comment:
Class names should represent the meaning of the content. Instead of 'nav_link is-active', consider using 'nav__link nav__link--active' to follow BEM methodology.